setup-ovs.sh

```diff
@@ -67,7 +75,17 @@ if [[ $(nmcli conn | grep -c ovs) -eq 0 ]]; then
       nmcli c delete $(nmcli c show |grep ovs-cnv |awk '{print $1}') || true
   else
       nmcli conn mod brcnv-iface connection.autoconnect yes
+      nmcli conn up ovs-slave-$secondary_device
+      # ocp 4.7
```

Comment:
perhaps we would do better wit using the `path` in networkManager to check what the folder is (we need to ack this with them of course). then we could just get the folder name like this:
```
NetworkManager --print-config | sed -nr "/^\[keyfile\]/ { :l /^path[ ]*=/ { s/.*=[ ]*//; p; q;}; n; b l;}"
```

Reply:
Yep, we will do that (what we talked offline)

Reply:
I am adding this to the test scripts in my lab to test, in all situation 
I will start with 4.7.19 until 4.8.4 to verify that it is not broken in later versions, Once it will work I will add it to the PR

Reply:
Added to the script, tested and its working well